CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ Every user-facing change should have an entry in this changelog. Please respect
 
 ## Unreleased
 
+- [Fix] Fix studio logout issue. (by @Alec4r) [PR #680](https://github.com/overhangio/tutor/pull/680)
```

Comment:
FYI there is no need to link to the PR. This forces you to open a PR before you can submit the actual change. That's ok, I'll remove the link in a later commit.